src/sync/branches.ts

```diff
@@ -2,9 +2,10 @@ import transformBranches from "./transforms/branch";
 import { getBranches as getBranchesQuery } from "./queries";
 import { GitHubAPI } from "probot";
 import { Repository } from "../models/subscription";
+import GitHubClient from "../github/client/github-client";
 
 // TODO: better typings
-export default async (github: GitHubAPI, repository:Repository, cursor?:string | number, perPage?:number) => {
+export default async (github: GitHubAPI, _newGithub: GitHubClient, _useNewGHClient: boolean, repository:Repository, cursor?:string | number, perPage?:number) => {
```

Comment:
Can you explain what these 2 new attributes are doing?

Reply:
Nothing right now. Next PRs will start using these similar to `pull-request` processor. 
The processors are overloaded so I couldnt get away with adding it only to that file hence the `_` infront to avoid linting errors.

Reply:
then maybe remove this change from this PR for now?

Reply:
Not that straight forward. Need to have these in the parameter list as they are required for a processor. Will try to refactor it but they will soon be needed when I work on `branch` and `commits`

Reply:
I'm just not a fan that we're polluting the function signature with a bunch of stuff that's inconsequential to it.

Personally, I would have preferred if we made a Github service abstraction so we can control the client at the service level, so long as the return signature is the same...